analysis/hu_zebrafish_linnaeus/README.rst

```diff
@@ -1,17 +1,21 @@
-Reproducing the zebrafish analysis
-==================================
-The analysis of the zebrafish data is a two-step process; the first step is the calculation of transition matrices using
-moslin, and the second is the downstream analysis of these transition matrices.
+zebrafish analysis
+===============
+
+The analysis of the zebrafish data is a two-step process:
+
+1. Calculation of transition matrices using moslin.
+2. Downstream analysis of the transition matrices.
 
-We provide `code <https://github.com/theislab/moslin/blob/analysis/simulations/analysis/hu_zebrafish_linnaeus/run_sbatch_hu_moslin.py>`_
-to obtain the couplings between hearts at consecutive time points on a SLURM based cluster scheduling system.
 The input data files can be found on figshare: `scRNA-seq <https://figshare.com/account/projects/163357/articles/22494529>`_
```

Comment:
I'm not sure how stable these links are. The safest option would be to refer to the collection, which has a stable doi: https://doi.org/10.6084/m9.figshare.c.6533377.v1